bin/reth/src/db/mod.rs

```diff
@@ -147,13 +154,13 @@ impl<'a, DB: Database> DbTool<'a, DB> {
     /// Lists the given table data
     fn list(&mut self, args: &ListArgs) -> Result<()> {
```

Comment:
It'd be really nice if you could make the `list` fn somehow return a HashMap of maybe generic return values, and then convert the return value to a table, vs printing the table inline. that would allow us to use it in other places, e.g. if we wanted to build an interactive TUI which renders the output of `list`

Reply:
Changed this up- let me know if this is what you were looking for 😄